receiver/otlpreceiver/config.go

```diff
@@ -32,6 +32,9 @@ type HTTPConfig struct {
 
 	// The URL path to receive logs on. If omitted "/v1/logs" will be used.
 	LogsURLPath string `mapstructure:"logs_url_path,omitempty"`
+
+	// The URL path to receive profiles on. If omitted "/v1/profiles" will be used.
+	ProfilesURLPath string `mapstructure:"profiles_url_path,omitempty"`
```

Comment:
I can't find the `/v1/profiles` part on the spec anywhere. I would prefer to remove this option for now to not have anything profiling specific in the configuration until profiling is more stable

Reply:
Sure, I can remove the option. But how would the HTTP handler decide which endpoint it listens to?

Reply:
Also, the only place where signal paths are specificed is for SDK exporters, which doesn't make much sense to add profiles to.

Reply:
> But how would the HTTP handler decide which endpoint it listens to?

We can hardcode `/v1/profiles` for now if that's what the profiling SIG believes will end up being used?

> Also, the only place where signal paths are specificed is for SDK exporters, which doesn't make much sense to add profiles to.

The OTLP specification mentions it for all other signals https://opentelemetry.io/docs/specs/otlp/#otlphttp-request

Reply:
Ah, I was looking in the spec, and the OTLP spec is in the proto repository 🤦 
I have made the config static.